.golangci.apis.yml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 run:
-  timeout: 3m
+  timeout: 10m
```

Comment:
We are getting timeouts quite often here, so increased the timeout.

https://public-prow.loodse.com/view/gs/prow-dev-public-data/pr-logs/pull/kubermatic_kubermatic/14114/pre-kubermatic-lint/1887436863854088192